x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/persistence/AnomalyDetectorsIndex.java

```diff
@@ -5,6 +5,7 @@
  */
 package org.elasticsearch.xpack.core.ml.job.persistence;
 
+import org.elasticsearch.ExceptionsHelper;
```

Comment:
Do you think it is beneficial to only ever use the `org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper`? Flip-flopping between the two in the code base has caused confusion for myself as I always expect it to be ours with all of our methods.

Reply:
Agreed, that's why I added a copy of `unwrapCause` in the ML version. I'll remove the other one in files of this PR where both are uses.